src/Api/Auth/Models/Response/PendingOrganizationAuthRequestResponseModel.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.Reflection;
+using Bit.Core.Auth.Models.Data;
+using Bit.Core.Models.Api;
+
+namespace Bit.Api.Auth.Models.Response;
+
+public class PendingOrganizationAuthRequestResponseModel : ResponseModel
+{
+    public PendingOrganizationAuthRequestResponseModel(OrganizationAdminAuthRequest authRequest, string obj = "pending-org-auth-request") : base(obj)
+    {
+        if (authRequest == null)
+        {
+            throw new ArgumentNullException(nameof(authRequest));
+        }
+
+        Id = authRequest.Id.ToString();
+        UserId = authRequest.UserId.ToString();
+        OrganizationUserId = authRequest.OrganizationUserId.ToString();
+        Email = authRequest.Email;
+        PublicKey = authRequest.PublicKey;
+        RequestDeviceIdentifier = authRequest.RequestDeviceIdentifier;
+        RequestDeviceType = authRequest.RequestDeviceType.GetType().GetMember(authRequest.RequestDeviceType.ToString())
```

Comment:
Is this how we display the pretty device type in other places, I would have thought we just return the enum value and the front end is responsible for translating that to the pretty name.

Reply:
That's how the `AuthRequestResponseModel` was handling the device type, so I repeated that functionality here as well. Not sure if we have a preference, but if it is to handle it in the front end I can make that change and just return the enum value.

Reply:
Fair enough, it's fine then.